modules/Bio/EnsEMBL/Compara/PipeConfig/DumpMultiAlign_conf.pm

```diff
@@ -229,6 +229,18 @@ sub pipeline_analyses {
             -hive_capacity => 15,
             -rc_name => 'crowd_with_reg_conf',
             $self->o('mode') eq 'tar' ? () : ( -flow_into => [ 'compress' ] ),
+            -flow_into      => { '-1' => 'dumpMultiAlign_8GB', }, 
+        },
+        {  -logic_name    => 'dumpMultiAlign_8GB',
+            -module        => 'Bio::EnsEMBL::Compara::RunnableDB::DumpMultiAlign::DumpMultiAlign',
+            -parameters    => {
+                'cmd' => [ 'perl', '#dump_program#', '--species', '#species#', '--mlss_id', '#mlss_id#', '--masked_seq', $self->o('masked_seq'), '--split_size', '#split_size#', '--output_format', '#format#', '--output_file', '#output_dir#/#base_filename#.#region_name#.#format#' ],
+                'output_file_pattern' => '#output_dir#/#base_filename#.#region_name##filename_suffix#.#format#',
+                'reg_conf'      => $self->o('reg_conf'),
+            },
+            -hive_capacity => 15,
+            -rc_name => 'crowd',
```

Comment:
I said it wrong. You're right; crowd_with_reg_conf is 2Gb, and so is crowd in fact. Maybe you have changed your crowd class to be 8Gb ?
Anyway, the important thing is that the the resource class for the high-mem analysis should include the registry file (see how crowd is defined and copy that)